core/templates/dev/head/pages/topic_editor/questions/questions_tab_directive.html

```diff
@@ -89,7 +93,7 @@ <h3 class="form-heading" ng-if="questionSuggestionThreads.length > 0"> Questions
 
   <question-editor question-id="activeQuestion.getId()"
                    misconceptions="emptyMisconceptionsList"
```

Comment:
I know this was there before itself, but does QuestionsTabDirective have this variable?

Reply:
I added it to QuestionsTabDirective